__init__.py

```diff
@@ -191,8 +191,11 @@ def main():
             importer = CommandLineCraftImporter()
             result = importer.execute(bpy.context, args.input_file, args.colliders, args.allow_no_material_mesh)
         else:
-            importer = CommandLineImporter()
-            result = importer.execute(bpy.context, args.input_file, args.colliders)
+            try:
+                importer = CommandLineImporter()
+                result = importer.execute(bpy.context, args.input_file, args.colliders)
+            except EOFError:
+                print('ERROR: the .mu file is invalid')
```

Comment:
This is a different codepath. Here, we are in the code path for single .mu files. This is because it is caught in import_craft (and not in import_mu) that we also need to catch it here.